libraries/r-3-runtime/context/pipe

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+ROOTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
+
+CRAN_MIRROR=${CRAN_MIRROR:=http://cran.rstudio.com/}
+
+echo "options(repos=structure(c(CRAN=\"$CRAN_MIRROR\")))" > ~/.Rprofile
```

Comment:
You could have added this as an artifact and then we wouldn't need the pipe.r file at all. `pipe` would just be the `pipe.r` file with the right `#!` line at the top to make it execute as r